examples/registry_example/swift_deps.bzl

```diff
@@ -0,0 +1,8 @@
+load("@rules_swift_package_manager//swiftpkg:defs.bzl", "swift_registry_package")
+
+def swift_dependencies():
+    swift_registry_package(
+        name = "swiftpkg_apple.swift_collections",
+        dependencies_index = "@//:swift_packages_index.json",
+        url = "https://artifactory.global.square/artifactory/api/swift/swift-test/apple/swift-collections/1.1.0.zip",
```

Comment:
This should be a public URL so we can test in CI

Reply:
> This should be a public URL so we can test in CI

This will be tough, because there's no public registry out there. We might be able to create a mock registry in python, and spin that up on CI during tests?

Reply:
I just ended up removing this example altogether, since it really only works on my machine.